src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -875,6 +904,12 @@ type fetchTaggedResultsIter struct {
 	tagEncoder      serialize.TagEncoder
 	iOpts           instrument.Options
 	instrumentClose func(error)
+	startTime       time.Time
```

Comment:
where is this set?

Reply:
also what's the difference between this and `fetchStart`?

Reply:
ugh it's not. clearly a test is needed for the metrics. 

renamed this to `dataReadStart` to make it more clear.